<?xml version="1.0" encoding="utf-8" ?>
<launch>
  <group ns="ros_gpio">
    <node pkg="ros_gpio" name="gpio_server" type="gpio_server" />
    <node pkg="ros_gpio" name="test_ros_gpio_client" type="test_ros_gpio_client" args="20" />
  </group>
</launch>
